examples/C/src/modal_models/FurutaPendulum/FurutaPendulum.lf

```diff
@@ -16,7 +16,7 @@
 target C {
   timeout: 3 secs,
   fast: true,
-  flags: "-lm",
+  compiler-flags: "-lm",
```

Comment:
```suggestion
```

Reply:
Well, for these examples, I suggest just commenting out the offending lines.